cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/tasks/CassandraState.java

```diff
@@ -640,6 +644,10 @@ public void update(Protos.TaskStatus status) throws IOException {
                     }
 
                     update(cassandraTask);
+                    if (cassandraTask.isTerminated()) {
+                        final String metricSuffix = this.getClass().getName() + "." + cassandraTask.getName();
```

Comment:
Ok, but the `metricsSuffix` is still something like `CassandraState.node-0` which doesn't say to me a metric for reporting task failures.